dashboard/d_graphs.py

```diff
@@ -264,7 +265,7 @@ def visualise_heatmap(data_df: pd.DataFrame, by_title: bool, colourscheme: str =
             field='week_num', order='ascending')),
         y=alt.Y('weekday:O', title='Day of the Week',  sort=WEEKDAY_ORDER),
         color=alt.Color(f'{vals}:Q', title='Polarity Score',
-                        scale=alt.Scale(scheme=colourscheme)),
+                        scale=alt.Scale(scheme=colourscheme, domain=[-1, 1])),
```

Comment:
I think fixing this colour gradient will make the graphs a lot clearer to understand, thank you :)